diff --git a/build/pdf.mjs b/build/pdf.mjs
index a33949b08e35e2edf09321c382b9ee34b3a56a8d..41c4442553de2d15c611cfef7957d69318a03963 100644
--- a/build/pdf.mjs
+++ b/build/pdf.mjs
@@ -13969,7 +13969,7 @@ class TextLayer {
     div.style.position = "absolute";
     div.textContent = "X";
     document.body.append(div);
-    this.#minFontSize = div.getBoundingClientRect().height;
+    this.#minFontSize = Math.max(1, div.getBoundingClientRect().height);
     div.remove();
   }
   static #getAscent(fontFamily, style, lang) {
